app/assets/javascripts/controllers/network_router/network_router_form_controller.js

```diff
@@ -41,6 +46,30 @@ ManageIQ.angular.app.controller('networkRouterFormController', ['$http', '$scope
     miqService.miqAjaxButton(url, $scope.networkRouterModel, { complete: false });
   };
 
+  $scope.addInterfaceClicked = function() {
+    miqService.sparkleOn();
+    var url = '/network_router/add_interface/' + networkRouterFormId + '?button=addInterface';
```

Comment:
Why `addInterface` when the info is already contained in the path? This should be just:
```javascript
var url = '/network_router/add_interface/' + networkRouterFormId + '?button=add';
```